avocado/plugins/run.py

```diff
@@ -205,7 +205,7 @@ def run(self, args):
         try:
             args.job_timeout = time_to_seconds(args.job_timeout)
         except ValueError as e:
-            LOG_UI.error(e.message)
+            LOG_UI.error(e.args[0])
```

Comment:
This would fail in case of plain `raise ValueError`. How about: `e.args[0] if e.args else "ValueError"`?

Reply:
Well since we only use `time_to_seconds` and this is the only little issue I found, let's ignore it until such situation happens...